packages/react-core/src/components/FileUpload/FileUploadField.tsx

```diff
@@ -78,6 +78,10 @@ export interface FileUploadFieldProps extends Omit<React.HTMLProps<HTMLDivElemen
   containerRef?: React.Ref<HTMLDivElement>;
   /** Text area text changed */
   onTextChange?: (text: string) => void;
+  /** Callback for when focus is lost on the TextArea field */
```

Comment:
Nit.  I would just say "text area". instead of TextArea here and in the other prop description.